### DISCUSSION TITLE
Ent rolling restarts with Einhorn; still recommended?

### DISCUSSION THREAD

**Discussion by ericburns** [sidekiq/sidekiq#5289] [Q&A] [answered]
I'm looking into rolling restarts with Sidekiq Ent. In the [wiki](https://github.com/mperham/sidekiq/wiki/Ent-Rolling-Restarts), it is recommended to use [Einhorn](https://github.com/stripe-archive/einhorn). Einhorn appears to be archived and in read-only mode. Is this still the suggested path for rolling restarts?

Are there alternatives?

**Comment by mperham** [comment, marked as answer]
Einhorn is considered “finished”. It might be read only but it performs its job without issue. At some point in the future we might need to fork it to provide maintenance but today it works well as is.
